src/acceptance/assets/app/go_app/Makefile

```diff
@@ -51,11 +54,31 @@ start: build
 deploy: build
 	./deploy.sh
 
-generate: go.mod ../../../../../api/custom-metrics-api.yaml
+openapi-spec-path := ../../../../../api
+openapi-specs := $(wildcard ${openapi-spec-path}/*.openapi.yaml)
+openapi-specs-bundled-path := ./internal/openapi-specs.bundled
+
+bundled-openapi-specs: ${bundled-openapi-specs-list}
+bundled-openapi-specs-list := \
+        ${openapi-specs-bundled-path}/application-metric-api.openapi.bundled.yaml \
+        ${openapi-specs-bundled-path}/custom-metrics-api.openapi.bundled.yaml \
+        ${openapi-specs-bundled-path}/policy-api.openapi.bundled.yaml \
+        ${openapi-specs-bundled-path}/scaling-history-api.openapi.bundled.yaml
+
+# # Treat these files as intermediate ones and don't rebuild everything
+# # just because they are missing.
+.SECONDARY: ${bundled-openapi-specs-list}
+
+${bundled-openapi-specs-list}: $(patsubst %.openapi.bundled.yaml,${openapi-spec-path}/%.openapi.yaml,$(notdir $@))
+	swagger-cli bundle \
+		$(patsubst %.openapi.bundled.yaml,${openapi-spec-path}/%.openapi.yaml,$(notdir $@)) \
+		--outfile '$@'
+
```

Comment:
I think this not needed as the generator supports remote references via an undocumented flag:
```
--allow-remote
```

https://github.com/ogen-go/ogen/blob/2e4b8dca0a88a88f9dea8eecefe24bfeb55ef27b/cmd/ogen/main.go#L149